packages/svelte/src/store/index.js

```diff
@@ -16,8 +18,11 @@ const subscriber_queue = [];
  * @returns {import('./public').Readable<T>}
  */
 export function readable(value, start) {
+	const w = writable(value, start);
 	return {
-		subscribe: writable(value, start).subscribe
+		// @ts-expect-error we don't want this to be on the public types
+		[SUBSCRIBERS_SYMBOL]: w[SUBSCRIBERS_SYMBOL],
```

Comment:
Should this be made non-enumerable then?

Reply:
`Object.assign` copies enumerable symbols as does object spreading.

Reply:
I think it's _probably_ harmless — people aren't really spreading or assigning the results of calling `writable` and `readable`, and there's not much damage that could be caused by people inadvertently getting a reference to the subscriber list. Feels like overkill to use `Object.defineProperty` here